cli/command/context/create.go

```diff
@@ -65,7 +65,8 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 	return cmd
 }
 
-func runCreate(cli command.Cli, o *createOptions) error {
+// RunCreate creates a Docker context
+func RunCreate(cli command.Cli, o *createOptions) error {
```

Comment:
I don't see a way for an external caller of `RunCreate` to produce a `createOptions`. OTOH the same seems to be true of e.g. `RunPush` and `RunSave` so maybe thesre's something I'm missing...